addons/event_stand/models/event_stand_type.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class EventStandType(models.Model):
+    _name = 'event.stand.type'
+    _description = 'Event Stand Type'
+    _inherit = ['image.mixin']
+
+    name = fields.Char('Name', required=True)
+    event_id = fields.Many2one('event.event', string='Event')
+    event_type_id = fields.Many2one('event.type', string='Event Category', ondelete='cascade')
+    can_be_shared = fields.Boolean(
```

Comment:
I don't understand this option... I remember we talked about it but still unclear.